models/qwen3_14b/paged_attention_pypto.py

```
@@ -0,0 +1,816 @@
+                        target_memory=pl.MemorySpace.Vec,
+                    )
+                )
+                s0_scaled = pl.tile.muls(s0, SCALE)
+                valid_cols0 = pl.min(BLOCK_SIZE, seq_len)
+                s0_valid = pl.set_validshape(s0_scaled, lane_rows, valid_cols0)
+                s0_filled = pl.fillpad(s0_valid, pad_value=pl.PadValue.min)
+                s0_masked = pl.set_validshape(s0_filled, lane_rows, BLOCK_SIZE)
+                zrow = pl.tile.muls(pl.row_max(s0_masked, tmp), 0.0)
+                one = pl.tile.adds(zrow, 1.0)
+                m = pl.tile.adds(zrow, -3.0e38)
+                l = zrow
```

Ruff (0.16.1)

[error] Ambiguous variable name: `l`

(E741)